whois-api/src/test/java/net/ripe/db/whois/api/util/TestUtil.java

```diff
@@ -0,0 +1,31 @@
+package net.ripe.db.whois.api.util;
+
+
+import net.ripe.db.whois.common.rpsl.RpslObject;
+
+public class TestUtil {
+    public static final String AS102_STRING = "" +
```

Comment:
Please don't move the test data out of the test class, this is less readable. I'm fine with some duplication if it's more readable, and can be modified in a test class without side-effects elsewhere.